Dockerfile

```diff
@@ -21,9 +20,22 @@ RUN npm run build
 
 # Deployment Stage
 FROM $BASE_IMAGE as deployment
-RUN apk add libexpat=2.6.0-r0  \
+RUN apk add libexpat=2.6.2-r0  \ 
     && rm -rf /var/cache/apk/*
 
+
 COPY --from=build /app/dist/ /usr/share/nginx/html/
 RUN rm /etc/nginx/conf.d/default.conf
-COPY nginx-vue.conf /etc/nginx/conf.d/
\ No newline at end of file
+COPY nginx-vue.conf /etc/nginx/conf.d/
+
+RUN chown -R nginx:nginx /var/cache/nginx /var/run /var/log/nginx /usr/share/nginx/html && \
```

Comment:
For more information about the permissions, please check:
https://ticketsystem.dbildungscloud.de/browse/DBP-620